libs/GraphicECS/SFML/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+set(GRAPHIC_SRC_COMPONENT_ROOT ${PROJECT_SOURCE_DIR}/libs/GraphicECS/SFML/Components)
+set(GRAPHIC_SRC_RESOURCE_ROOT ${PROJECT_SOURCE_DIR}/libs/GraphicECS/SFML/Resources)
+set(GRAPHIC_SRC_SYSTEM_ROOT ${PROJECT_SOURCE_DIR}/libs/GraphicECS/SFML/Systems)
+
+set(TARGET ${PROJECT_NAME}_SFMLGraphicECS)
+
+add_subdirectory(Systems)
+add_subdirectory(Components)
+add_subdirectory(Resources)
+
+set(SRC
+    ${SRC_GRAPHIC_COMPONENT}
+    ${SRC_GRAPHIC_RESOURCE}
+    ${SRC_GRAPHIC_SYSTEM}
+)
+
+add_library(${TARGET} SHARED ${SRC})
+
+target_include_directories(${TARGET}
+    PRIVATE ${CONAN_INCLUDE_DIRS_SFML}
+    PUBLIC ${GRAPHIC_SRC_COMPONENT_ROOT}
+    PUBLIC ${GRAPHIC_SRC_RESOURCE_ROOT}
+    PUBLIC ${GRAPHIC_SRC_SYSTEM_ROOT}
```

Comment:
Public ? Private will be more appropriate

Reply:
No Public is good, because other's part need these path